crates/swc_ecma_loader/tests/tsc_resolver.rs

```diff
@@ -97,6 +97,46 @@ fn pattern_1() {
     }
 }
 
+#[test]
+fn base_url_precedence() {
```

Comment:
The test name says that it's a test about precedence, but it's not testing precedence. Can you add some files to https://github.com/swc-project/swc/tree/039c1685c663cd138e4a0cf769b506d1fd419ba5/crates/swc_ecma_loader/tests and check if it's actually precedence?

Reply:
I'vs updated test cases into two cases for more clear. 😊